tools/pd-simulator/simulator/cases/add_nodes.go

```diff
@@ -34,11 +34,8 @@ func newAddNodes() *Case {
 
 	for i := 1; i <= storeNum; i++ {
 		simCase.Stores = append(simCase.Stores, &Store{
-			ID:        IDAllocator.nextID(),
-			Status:    metapb.StoreState_Up,
-			Capacity:  1 * units.TiB,
-			Available: 900 * units.GiB,
-			Version:   "2.1.0",
```

Comment:
I think we need some test to ensure release version is available for these cases

Reply:
The store version can be configured through config file.

Reply:
whar is its default version